opentelemetry-jaeger/src/lib.rs

```diff
@@ -211,8 +211,23 @@ impl trace::SpanExporter for Exporter {
     fn export(&self, batch: Vec<Arc<trace::SpanData>>) -> trace::ExportResult {
         match self.uploader.lock() {
             Ok(mut uploader) => {
-                let jaeger_spans = batch.into_iter().map(Into::into).collect();
-                uploader.upload(jaeger::Batch::new(self.process.clone(), jaeger_spans))
+                let mut jaeger_spans: Vec<jaeger::Span> = Vec::new();
+                let mut process_tags: Vec<jaeger::Tag> = Vec::new();
+
+                for span in batch.iter() {
```

Comment:
Could just use the `batch.into_iter()` and switching the following two line like 
```rust
process_tags.extend(build_process_tags(&span));
jaeger_spans.push(span.into());
```